frontend/public/src/components/ProgramCourseInfoCard.js

```diff
@@ -80,38 +75,6 @@ export class ProgramCourseInfoCard extends React.Component<ProgramCourseInfoCard
             </div>
           </div>
         </div>
-        <div className="row flex-grow-1 pt-3">
```

Comment:
Bit too much - the row and column need to stick around or the course image looks weird:
![image](https://user-images.githubusercontent.com/945611/211391099-ea432d75-3068-4907-a82a-32d1705085f9.png)
The top card has the row and the column but nothing in it.